### DISCUSSION TITLE
Welcome to bee-queue Discussions!

### DISCUSSION THREAD

**Discussion by compwright** [bee-queue/bee-queue#551] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jorenvandeweyer** [comment]
Hi there 👋

I'd like to open the discussion with a suggestion to rewrite the package in TypeScript. The code base is relatively small, so rewriting this should be low effort.
